annet/annlib/netdev/devdb/data/devdb.json

```diff
@@ -67,6 +65,9 @@
     "Huawei.EI": "-EI(-|$)",
     "Huawei.HI": "-HI(-|$)",
     "Huawei.SI": "-SI(-|$)",
+    "Huawei.OptiXtrans": " DC\\d+",
```

Comment:
I would suggest to remove \d+ match here and add optional OptiXtrans.  This way we would support more variants of possible naming scheme.

```diff
% git diff
diff --git a/annet/annlib/netdev/devdb/data/devdb.json b/annet/annlib/netdev/devdb/data/devdb.json
index 022a03f..7fc53a6 100644
--- a/annet/annlib/netdev/devdb/data/devdb.json
+++ b/annet/annlib/netdev/devdb/data/devdb.json
@@ -65,8 +65,8 @@
     "Huawei.EI": "-EI(-|$)",
     "Huawei.HI": "-HI(-|$)",
     "Huawei.SI": "-SI(-|$)",
-    "Huawei.OptiXtrans": " DC\\d+",
-    "Huawei.OptiXtrans.DC": " DC\\d+",
+    "Huawei.OptiXtrans": " OptiXtrans|DC",
+    "Huawei.OptiXtrans.DC": " DC",
     "Huawei.OptiXtrans.DC.DC908": " DC908",
 
     "Juniper": "^[Jj]uniper",
```
```
% cat t.py 
from annet.hardware import HardwareView

for x in [
    'HardwareView("Huawei OptiXtrans", "")',
    'HardwareView("Huawei DC", "")',
    'HardwareView("Huawei OptiXtrans DC", "")',

    'HardwareView("Huawei DC908", "")',
    'HardwareView("Huawei OptiXtrans DC908", "")',

    'HardwareView("Huawei DC100500", "")',
    'HardwareView("Huawei OptiXtrans DC100500", "")',
]:
    print("========================================")
    print(f"{x}")
    print(eval(x))
```
```
% python3 t.py 
========================================
HardwareView("Huawei OptiXtrans", "")
Huawei | Huawei.OptiXtrans | OptiXtrans
========================================
HardwareView("Huawei DC", "")
DC | Huawei | Huawei.DC | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | OptiXtrans | OptiXtrans.DC
========================================
HardwareView("Huawei OptiXtrans DC", "")
DC | Huawei | Huawei.DC | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | OptiXtrans | OptiXtrans.DC
========================================
HardwareView("Huawei DC908", "")
DC | DC.DC908 | DC908 | Huawei | Huawei.DC | Huawei.DC908 | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | Huawei.OptiXtrans.DC.DC908 | Huawei.OptiXtrans.DC908 | OptiXtrans | OptiXtrans.DC | OptiXtrans.DC.DC908 | OptiXtrans.DC908
========================================
HardwareView("Huawei OptiXtrans DC908", "")
DC | DC.DC908 | DC908 | Huawei | Huawei.DC | Huawei.DC908 | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | Huawei.OptiXtrans.DC.DC908 | Huawei.OptiXtrans.DC908 | OptiXtrans | OptiXtrans.DC | OptiXtrans.DC.DC908 | OptiXtrans.DC908
========================================
HardwareView("Huawei DC100500", "")
DC | Huawei | Huawei.DC | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | OptiXtrans | OptiXtrans.DC
========================================
HardwareView("Huawei OptiXtrans DC100500", "")
DC | Huawei | Huawei.DC | Huawei.OptiXtrans | Huawei.OptiXtrans.DC | OptiXtrans | OptiXtrans.DC
```